src/main/java/teammates/ui/template/InstructorFeedbackSessionActions.java

```diff
@@ -66,6 +70,7 @@ public InstructorFeedbackSessionActions(PageData data, FeedbackSessionAttributes
 
         this.publishButton = new FeedbackSessionPublishButton(data, session, returnUrl, instructor,
                                                               PUBLISH_BUTTON_TYPE);
+        this.isAllowedToEmailPublish = session.isPublished() && session.isPublishedEmailEnabled();
```

Comment:
I think we might want to allow sending the emails even if the published email setting was not enabled? Need to check with @damithc though.

Reply:
@damithc your thought?

Reply:
yes, we should allow it. The use case is the instructor doesn't want to send the email automatically but wants to send it manually instead.